wbooks/src/app/components/HeaderBackButton/index.tsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import { Image } from 'react-native';
+import icBack from '@assets/general/ic_back.png';
+
+import styles from './styles';
+
+function HeaderBackButton() {
+  return <Image style={styles.container} source={icBack} />;
```

Comment:
And the `resizeMode`? Maybe `cover`?

Reply:
I tried with some combinations using resizeMode, but the result was the same

Reply:
Mm if I add resizeMode="cover", I can not see any change

Reply:
Done!